biblio_test.py

```diff
@@ -0,0 +1,21 @@
+from numpy import *
```

Comment:
recommend `from __future__ import print_function` and then fix all prints to keep this Py3 compatible (what I use).